support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -170,4 +172,14 @@ public String getId() throws AbortException {
         }
     }
 
+    @Whitelisted
+    public @Nonnull String getAbsoluteUrl() throws AbortException {
+        String root = Jenkins.getActiveInstance().getRootUrl();
+        if (root == null) {
+            throw new AbortException("Jenkins root url is not configured!");
+        } else {
+            return root + build().getUrl();
```

Comment:
@rsandell It's a special deprecation just to avoid using it in jelly AFAIK. Anyway, the restriction on sandboxed scripts is totally justifying this new API.